spec/defines/auth_param_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe 'squid::auth_param' do
+  let :pre_condition  do
+    ' class{"::squid":
+        config => "/tmp/squid.conf"
+      }
+    '
+  end
+  let(:title) { 'auth' }
+  context 'when parameters are set' do
+
+    entries = ['program /usr/lib64/squid/basic_ncsa_auth /etc/squid/.htpasswd',
+     'children 5',
+     'realm Squid Basic Authentication',
+     'credentialsttl 5 hours']
+
+    let(:params) do
+      {  scheme: 'basic',
```

Comment:
This will fail robocop as it it, you need to do the same as I just did to my files.